.github/workflows/main.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     strategy:
       matrix:
         rust: [stable]
-        python-version: ['3.7.16', 3.8, 3.9, "3.10", "3.11"]
+        python-version: ['3.7.16', 3.8, 3.9, "3.10", "3.11", "3.12.0-rc.3"]
```

Comment:
I assume we'll have to wait a little bit for the default gha image to get 3.12.0.

Reply:
It's not in https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json, but the ABI should be identical to the released one